src/Accuracy.js

```diff
@@ -1,14 +1,35 @@
-import React, { useContext } from "react";
+import React, { useCallback, useContext, useEffect, useState } from "react";
 import { utilityContext } from "./utilities/utilityContext";
+import text from "./resources/content";
 import "./Accuracy.css";
 
 export default function Accuracy() {
-  const { numLetterPassed, numLetterFailed,lengthOfText } = useContext(utilityContext);
+  const { pressedKeys } = useContext(utilityContext);
+  const [numLetterFailed, setNumLetterFailed] = useState(0);
+  const [numLetterPassed, setNumLetterPassed] = useState(0);
+
+  const currentIndex = useCallback(
```

Comment:
This is a function? why use it as a variable?

Reply:
was trying out that syntax